### DISCUSSION TITLE
数式編集時、プレビューが反応しないことがある

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#313] [要望・アイデア・不具合]
投稿の編集中に、エディタ側で数式だけを修正したとき、プレビュー側のすべての数式部分の表示が抜け落ちることがあります。

　このような時でも、数式以外の文章部分に適当な一文字を書き込むだけで、数式の表示は復活します（そのあと、余計な一文字はもちろん削除します）。したがって、致命的な問題というほどではありませんが、余計な手間がかかって面倒です。なんとかならないでしょうか？

　一文字打ち込む毎に、リアルタイムでプレビューを更新するという親切な仕様ですが、数式までには手が回っていないように見受けられます。

　そこで提案です。数式の表示の一時的な抜け落ちは我慢するとして、「プレビュー更新」ボタンを追加装備し、これをクリックするだけで、要りもしない文字を追加・削除する手間は不要になるようにしていただけると便利です。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご報告ありがとうございます🙏
 >プレビュー側のすべての数式部分の表示が抜け落ちることがあります。

「プレビュー更新」ボタンの追加はお約束できないのですが
数式関係は他にも不具合が報告されているため、今回いただいたものも合わせて対応します🙇
